manager/aica_django/settings.py

```diff
@@ -172,6 +172,8 @@
 CELERY_ACCEPT_CONTENT = ["json"]
 
 CELERY_ENABLE_UTC = True
+CELERY_SEND_EVENTS = True
```

Comment:
What does this (and the following line) do/impact?

Reply:
These SEND EVENTS are needed for monitoring by tools like Celerymon/flower